apps/api/src/app/shared/validators.ts

```diff
@@ -0,0 +1,7 @@
+import { IsIn } from 'class-validator';
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export const IsBooleanAny = () =>
+  IsIn([true, false, 'true', 'false'], {
+    message: '$property must be a boolean value',
+  });
```

Comment:
Sure. I made this validator because frontend sends "true"/"false" for some fields - maybe we will change this behaviour to send only real boolean and use built-in `@IsBoolean`?

Reply:
but for example in some e2e tests for integration - regular boolean is sent.